packages/algolia/src/client.ts

```diff
@@ -72,15 +72,19 @@ export class AlgoliaSearchClient {
   }
 
   async searchEntity<T extends keyof EntityResponses>(
```

Comment:
to be fair I think the idea was that this method is used to search just a single entity. If you want to change it thats fine but in that case its probably best to rename to either `searchEntities` or just `search`

Reply:
I noticed that when we search across multiple entities

```typescript
const result = await client.searchEntity(
    ['external-author', 'research-output'],
    'asdas',
  );

  for (const record of result.hits) {
    if (record.__meta.type === "external-author") {
      record
    }
  }
```
the result contains hits which are a union of the selected entities, which is great, but we are also unable to easily tell them apart using the __meta.type. In fact I am able to change it to anything as long as its a key EntityResponses. Perhaps now its time to improve it?